docker/Dockerfile-github

```diff
@@ -0,0 +1,9 @@
+FROM jetscape/base:v1.3
+
+COPY --chown=jetscape-user:jetscape-user JETSCAPE-DOCKER /home/jetscape-user/JETSCAPE
+RUN cd JETSCAPE && \
+	mkdir build && cd build && \
+	cmake .. && make
```

Comment:
One more thing: We need to make sure we download and build all of the optional external packages, like is done here: https://github.com/JETSCAPE/JETSCAPE/blob/master/.github/workflows/test-build-external.yaml

Reply:
Right, I have not used git submodules before but perhaps this would be the "right" way to do it. The current treatment of external packages is just what we inherited -- I would suggest we use the current approach for the deploy container (you can basically just copy the linked code, should be trivial) and then for the future if you would like to propose to re-work using git submodules we can have a dedicated presentation/discussion at a COMP meeting, we will have to make sure all of the external packages are amenable to this. Sound ok?

Reply:
The change to git the external packages is done.